### PR TITLE
Use GitHub's forms for issue templates

### PR DESCRIPTION
**Pull request by BoD**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        You can also attach files by dragging them into the area.
+      value: |
+        ```
+        (Your logs here)
+        ```
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Anything else?
```
Comment (BoD) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:41]: This wasn't there in the original md one, but thought it could be useful

**Comment by martinbonnin** [comment]
Do you have a screenshot of how it looks like in real life?

**Comment by martinbonnin** [comment]
Ok, nevermind, I can try myself here 😊  https://github.com/BoD/apollo-kotlin

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  - type: textarea
+    attributes:
+      label: Logs
+      description: |
+        Paste logs and full stacktrace here.
+        You can also attach files by dragging them into the area.
+      value: |
+        ```
+        (Your logs here)
+        ```
```
Comment (martinbonnin) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:36]: This is a very nice touch 👌

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (martinbonnin) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:41]: 👍

**Review by martinbonnin — APPROVED** [review]
🤩

**Pull request merged**